### DISCUSSION TITLE
[Talk] PL/.NET with Todd Lewis on Aug 16th, 2024

### DISCUSSION THREAD

**Discussion by yrashk** [pgedc/pgedc#39] [General]
I am excited to announce that we will have a live talk on PL/.NET next Friday, Aug 16th, 2024 🥳 

São Paulo, Brazil: Fri Aug 16, 12:00 / 12:00 pm
San Francisco, California, USA: Fri Aug 16, 08:00 / 8:00 am
New York state, USA: Fri Aug 16, 11:00 / 11:00 am
London, UK: Fri Aug 16, 16:00 / 4:00 pm
Singapore: Fri Aug 16, 23:00 / 11:00 pm

The talk and the QA/discussion will be held over Zoom https://us06web.zoom.us/j/81017143773

---

pl/dotnet extends the PostgreSQL database to support stored procedures, functions, triggers, and DO blocks for the .NET platform, including both C# and F#. In our benchmark it is the fastest Procedural Language in PostgreSQL, and it has the widest range of unit testing. It natively supports 38 out of 46 standard user types, the widest range of any external Procedural Language in PostgreSQL. It is released as free software under the PostgreSQL (BSD-style) license. Our goal is to be the best Procedural Language in PostgreSQL and the best implementation of .NET stored procedures in any database.

In this talk, [Todd](https://www.linkedin.com/in/toddglewis2/) discusses the present and future of pl/dotnet, potential cooperation with other PLs and PostgreSQL extensions, and possible improvements in the PostgreSQL extension APIs.

---

Please indicate if you plan to attend by raising your hand. Let us know in the comments below if you'll be joining us!

**Comment by tlewis-ba** [comment]
I'm really looking forward to it!  Feel free to drop me questions ahead of time.

**Comment by yrashk** [comment]
Fellow @pgedc/members, this is your reminder that this talk is coming up this Friday! Please let us know if you are joining.

**Comment by brunoluizkatz-NETZSCH** [comment]
The call was recorded? If yes, here I can find the record?

**Reply by yrashk** [reply]
Yes, it was! I am preparing the recording and will publish it soon.

**Reply by yrashk** [reply]
I've published the (slightly edited) video recording here https://youtu.be/d0PPb5IuoU4
